packages/palette/src/elements/Select/Select.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { SFC } from "react"
 import styled, { css } from "styled-components"
 import {
+  maxWidth,
+  MaxWidthProps,
```

Comment:
I think we should have an option to set both `maxWidth` and `minWidth` here. And let clients figure out which one they want.

Reply:
makes sense! done